lib/resque/scheduler/delaying_extensions.rb

```diff
@@ -230,6 +135,20 @@ def get_last_enqueued_at(job_name)
 
       private
 
+      def enqueue_delayed(blob)
+        count = redis.zrem(:delayed_queue, [blob])
+        Resque::Scheduler.enqueue_from_config(decode_without_nonce(blob))
```

Comment:
I think this should techincally be `count.times { enqueue... }`, even though the caller always includes the nonce?